class/defaults.yml

```
@@ -2,3 +2,6 @@ parameters:
   appcat_service_postgresql:
     =_metadata: {}
     namespace: syn-appcat-service-postgresql
+    registry: pippo.com
+    image: appcat-service-postgresql
+    tag: v0.1
```

What's this registry? :laughing: 
According to https://syn.tools/syn/explanations/commodore-components/container-images.html we should maintain images a little bit differently:
```suggestion
    images:
      provider-postgresql:
        registry: ghcr.io
        image: vshn/appcat-service-postgresql
        tag: v0.1.0
```

---

It's a draft so yes will udpate with the right registry

---

Since we decided to use helm, we can leave the image details to [helm_values](https://syn.tools/syn/explanations/commodore-components/helm-charts.html#_example) dictionary, what do you think?

---

The current best practice for commodore component is to have this `images` key, and then to reference this in the `helm_values` via reclass. See this (incomplete) example: https://github.com/projectsyn/component-backup-k8up/blob/master/class/defaults.yml#L29